### DISCUSSION TITLE
ok, but whats that?

### DISCUSSION THREAD

**Discussion by Bernardoskau** [Squirrel/Squirrel.Windows#1773] [Q&A] [answered]
i was looking at the code in Visual Studio Code when i saw this link, i pressed ctrl, clicked on it and, i saw an website from asia with porn ads.
![Captura de Tela (25)](https://user-images.githubusercontent.com/95549506/144759975-4cda126c-e361-4af6-a6be-964a775fe83c.png)
![Captura de Tela (26)](https://user-images.githubusercontent.com/95549506/144759983-8e9dccdb-ce61-411a-8d04-401ff7929a29.png)
-

**Comment by robmen** [comment, marked as answer]
It looks like some extension to SVG's URL has been taken over. The reference seems to be unnecessary in the SVG, so I've removed it in #1774.

**Reply by Bernardoskau** [reply]
> It looks like some extension to SVG's URL has been taken over. The reference seems to be unnecessary in the SVG, so I've removed it in #1774.

Alright, thank you for the fast reply.
